.travis.yml

```diff
@@ -20,6 +20,7 @@ dist: trusty
 language: java
 
 jdk:
+  - openjdk7
   - openjdk8
```

Comment:
jdk8 JVM should be compatible with jdk7 java programs. This line maybe not necessary.

Reply:
Thanks for clarification. Remove L24